runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesOverlay.java

```diff
@@ -118,6 +118,10 @@ public Dimension render(Graphics2D graphics)
 
 			for (XpGlobe xpGlobe : xpChangedQueue)
 			{
+				if (config.disable99s() && xpGlobe.getCurrentLevel() == 99)
```

Comment:
Would it be better to instead prevent the globe from being added/updated in onExperienceChanged()?

Reply:
Will remove this, this was my initial way to get rid of it. I ended up moving it into the addXPGlobe() called in onExperienceChanged().